src/layout/src/components/CardSegment/CardSegment.js

```diff
@@ -62,6 +89,19 @@ const CardSegment = styled.div`
   flex: 1 0 auto;
   position: relative;
   ${getSegmentComputedStyles}
+
+  & & {
```

Comment:
Hm raised specificity of styles...is this for nested `CardSegments`? Would it be better to just move these border styles all together into the `CardSegment` 'cos right now these same border styles also exist in the parent `Card`. Perhaps with sibling selectors instead child selectors?

Reply:
Yes, this is for nested `CardSegment` components. Are you saying to remove the `& > ${CardSegment}` styles from `Card` and only have it in `CardSegment`? That'll probably work, haven't tested it.

"raised specificity of styles"... not sure what this means.

"Perhaps with sibling selectors instead child selectors"... I don't have much experience with sibling selectors, not sure how that would work.